webpack.config.js

```diff
@@ -87,5 +77,8 @@ module.exports = {
       amd: 'playkit-js',
       root: ['KalturaPlayer', 'core']
     }
+  },
+  optimization: {
+    minimize: PROD
```

Comment:
use `--mode production`/`--mode development` in package.json instead(for build/dev/watch scripts)

Reply:
done